pytorch_lightning/plugins/ddp_sequential_plugin.py

```diff
@@ -0,0 +1,403 @@
+        if not hasattr(model, "sequential_module") or not isinstance(model.sequential_module, nn.Sequential):
+            raise MisconfigurationException(
+                'Could not find a PipeLightningModule within the model. '
+                'Did you defined set your sequential model as an `sequential_module` attribute of your model ?')
+
+    def _find_and_init_pipe_module(self, model):
+        if hasattr(model, "sequential_module") and isinstance(model.sequential_module, LightningPipeModule):
+            # model has been wrapped already
+            return
+        elif hasattr(model, "sequential_module") and isinstance(model.sequential_module, nn.Sequential):
+            # try to wrap model for the user
+            model.sequential_module = LightningPipeModule(
+                model.sequential_module,
+                balance=self.balance,
+                microbatches=self.microbatches,
+                checkpoint=self.checkpoint,
+            )
+            # Update references for workers to access correct lightning functions when calling RPC
+            model.sequential_module.trainer = model.trainer
+            model.sequential_module.configure_optimizers = model.configure_optimizers
+
+            # Update references for main process to access correct lightning functions when calling RPC
+            model.sequential_module.module.model.trainer = model.trainer
+            model.sequential_module.module.model.configure_optimizers = model.configure_optimizers
+
+        else:
+            raise MisconfigurationException(
+                'Could not find a PipeLightningModule within the model. '
+                'Did you defined set your sequential model as an `sequential_module` attribute of your model ?'
+            )
+
+    def _assert_valid_model_balance(self, trainer):
+        model = trainer.get_model()
+        if sum(self.balance) != len(model.sequential_module):
+            raise MisconfigurationException(
+                f'The provided balance sum: {sum(self.balance)} does not'
+                f' match your Sequential length: {len(model.sequential_module)}')
+
+    def _skip_init_connections(self, trainer):
+        """
+        Skip initialization if torch is already initialized and we're in testing.
+        Returns: Whether to skip initialization
+
+        """
+        if torch_distrib.is_initialized() and trainer.testing:
+            return True
+        return False
+
+    def init_model_parallel_groups(self, trainer):
+        num_model_parallel = 1  # TODO currently no support for vertical model parallel
+        mpu.initialize_model_parallel(
+            model_parallel_size_=num_model_parallel,
+            pipeline_length=self.gpus_per_model
+        )
+
+    def _infer_check_num_gpus(self, trainer):
+        """
+        Infer the number of GPUs per model.
+        Args:
+            trainer: The trainer object.
+        Returns: The appropriate balance for the model
```

Comment:
```suggestion

        Args:
            trainer: The trainer object.

        Returns: The appropriate balance for the model
```
I guess that this separation shall be made everywhere for propper docs rendering...